converters/fromZigbee.js

```diff
@@ -1517,6 +1517,28 @@ const converters = {
             return {action: postfixWithEndpointName(`identify`, msg, model)};
         },
     },
+    LXN59_tuya_cover_options: {
+        cluster: 'closuresWindowCovering',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            if (msg.data.hasOwnProperty('tuyaMovingState')) {
+                const value = msg.data['tuyaMovingState'];
+                const movingLookup = {0: 'DOWN', 1: 'UP', 2: 'STOP'};
+                result.moving = movingLookup[value];
+            }
+            return result;
+        },
+    },
+    LXN59_cover_state_via_onoff: {
+        cluster: 'genOnOff',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.hasOwnProperty('onOff')) {
+                return {state: msg.data['onOff'] === 1 ? 'CLOSED' : 'OPEN'};
```

Comment:
`CLOSED` should be `CLOSE` (similar to other covers)

Reply:
ah sure you are right will correct that

Reply:
Done